.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,128 @@
+name: Build wheels
+
+on:
+  # Trigger on pull requests.
+  pull_request:
+
+  # Trigger on pushes to the mainline branches and version tags. This prevents building commits
+  # twice when the pull request source branch is in the same repository.
+  push:
+    branches:
+      - "maint"
+      - "master"
+    tags:
+      - "v*"
+
+  # Trigger on request.
+  workflow_dispatch:
+
+env:
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+jobs:
+  build_wheels:
+    name: Build wheel for ${{ matrix.os }}, Python ${{ matrix.pyver }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15] #, windows-2019]
+        pyver: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2.3.4
```

Comment:
This is more of a question, but what do these uses statements do exactly?

Reply:
They use scripts written by others:
https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions